tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -185,6 +185,10 @@ shader, this value can be used as desired.
 |                                        | (this is ``true`` in the Compatibility renderer,       |
 |                                        | ``false`` in Forward+ and Forward Mobile).             |
 +----------------------------------------+--------------------------------------------------------+
+| in bool **CLIP_SPACE_FAR**             | End of a clip space range.                             |
+|                                        | When using a Vulkan renderer, it's ``0``.              |
+|                                        | On compatibility renderer, it's ``-1``.                |
++----------------------------------------+--------------------------------------------------------+
```

Comment:
```suggestion
| in float **CLIP_SPACE_FAR**            | Clip space far ``z`` value.                  |
|                                        | In the Forward+ or Mobile renderers, it's ``0.0``. |
|                                        | In the Compatibility renderer, it's ``-1.0``.           |
+----------------------------------------+--------------------------------------------------------+
```
It's a `float`, not a `bool`. The ``z`` value should be specified.
Floats literals should be written with the decimal like `0.0` not `0`.
Use the names of each renderer. 

(The table formatting needs to be reformed, and apply these changes to the other two locations)

Reply:
ah, right, ty